js/runner_test.go

```diff
@@ -818,6 +818,81 @@ func TestVUIntegrationBlacklistScript(t *testing.T) {
 	}
 }
 
+func TestVUIntegrationBlockHostnamesOption(t *testing.T) {
+	r1, err := getSimpleRunner("/script.js", `
+					import http from "k6/http";
+					export default function() { http.get("https://k6.io/"); }
+				`)
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	hostnames := lib.HostnameTrie{}
+	if insertErr := hostnames.Insert("*.io"); !assert.NoError(t, insertErr) {
+		return
+	}
```

Comment:
```suggestion
	require.NoError(t, hostnames.Insert("*.io"))
```
`require` will abort the test and is much more readable (and hard to get wrong ;) ) IMO